airflow/dags/rt_loader/external_validation_service_alerts.sql

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This seems similar to the DAG tasks in `gtfs_schedule_history` in that it is creating an external table. Should we create a new DAG for creating these external RT tables where the DAG schedule run just once?

Reply:
I regret making `gtfs_schedule_history` its own DAG (or at the very least having it run once a day) for 2 reasons:

* when changes are made to the external table definition, they are not automatically picked up the next day (because the DAG only runs once). This has been a common source of errors after merging PRs :/. In general, what happens is the tasks depending on the table fail and then you have to go in and clear the once-off DAG tasks
* it seems helpful to keep the table creation close to the loading script. It's quick to see everything together. 

The payments_loader DAG follows a similar setup, where external tables are included in the loader DAG, just to keep everything together: 

![image](https://user-images.githubusercontent.com/2574498/143300387-6f50f74f-0f6c-472c-996e-a181f25cf485.png)

An alternate way to do it is to have a separate DAG for external tables (e.g. a rt_setup DAG) and have it set to `latest_only`. This it will run daily, but will skip if you clear it for days in the past.

Reply:
ok, yeah, I agree with what has been in the PR.